src/Diagnostics.ModelsAndUtils/Models/ResponseExtensions/ResiliencyReportData.cs

```diff
@@ -0,0 +1,226 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Text;
+
+namespace Diagnostics.ModelsAndUtils.Models.ResponseExtensions
+{
+    public class ResiliencyReportData : IResiliencyReportData
+    {
+        [JsonConverter(typeof(ResiliencyResource))]
+        ResiliencyResource[] resiliencyResourceList;
+
+        /// <summary>
+        /// Creates an instance of ResiliencyReportData
+        /// </summary>
+        public ResiliencyReportData(string customerName, ResiliencyResource[] resiliencyResourceList)
+        {
+            this.CustomerName = customerName;
```

Comment:
What if customer name is null or empty?

Reply:
Thank you! Changes it to:
            if (string.IsNullOrEmpty(customerName)) 
            {
                throw new ArgumentNullException(nameof(customerName), $"{nameof(customerName)} cannot be null or empty");
            }
            else
            {
                this.CustomerName = customerName;
            }